include/experimental/__p0009_bits/mdspan.hpp

```diff
@@ -339,6 +339,16 @@ class mdspan
     return static_cast<size_type>(__impl::__size(*this));
   };
 
+  MDSPAN_INLINE_FUNCTION constexpr bool empty() const noexcept {
+    return size() == 0;
```

Comment:
I think `empty()` has different preconditions than `size() == 0` -- it should be possible to call `empty()` even if the `size()` can't be computed, no?

Reply:
```c++
MDSPAN_INLINE_FUNCTION constexpr bool empty() const noexcept {
  for(std::size_t r = 0; r < rank(); ++r) {
    if(extent(r) == 0) {
      return true;
    }
  }
  return false;
}
```

Reply:
we defined it explicitly as Effects: Equivalent to return size() == 0

Reply:
ah wait that was changed in R1, uff this is gonna be poopy

Reply:
done